src/utils/testItemExporter.ts

```
@@ -0,0 +1,63 @@
+ */
+export function exportTestItem(
+  testItem: TestItem,
+  outputPath: string = './tmp/test-items.json'
+): void {
+  // Create output directory if it doesn't exist
+  const outputDir = dirname(outputPath);
+  if (!existsSync(outputDir)) {
+    mkdirSync(outputDir, { recursive: true });
+  }
+
+  let exportData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)